src/main/kotlin/com/hyperdevs/poeditor/gradle/PoEditorStringsImporter.kt

```diff
@@ -80,18 +82,41 @@ object PoEditorStringsImporter {
                               defaultLang: String,
                               resDirPath: String,
                               tags: List<String>?,
-                              languageValuesOverridePathMap: Map<String, String>?) {
+                              languageValuesOverridePathMap: Map<String, String>?,
+                              minimumTranslationPercentage: Int?) {
+        fun isPercentageTooLow(percentage: Double) = percentage < minimumTranslationPercentage ?: -1
+        fun List<ProjectLanguage>.joinAndFormat(transform: ((ProjectLanguage) -> CharSequence)) =
+            joinToString(separator = ", ", prefix = "[", postfix = "]", transform = transform)
```

Comment:
I'd move these functions outside of `importPoEditorStrings`, leaving them inside `PoEditorStringsImporter` as private functions